bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/internal/item/ItemResource.java

```diff
@@ -130,6 +131,7 @@
     @NonNullByDefault({})
     private DTOMapper dtoMapper;
     private final Set<ItemFactory> itemFactories = new HashSet<>();
+    private LocaleService localeService;
```

Comment:
Here you should have an error in your IDE: `The @NonNull field localeService may not have been initialized`
You can circumvent this by adding the `@NonNullByDefault({})` annotation here. This will basically exclude the field from the null annotation checks.